packages/python-packages/apiview-copilot/README.md

```diff
@@ -67,6 +61,27 @@ If you would like to search the knowledge base and see the output the way the LL
 
 This will dump the results to context.md which you can then view in VSCode with the preview editor.
 
+## Getting Comments from APIView
+
+If you need to retrieve comments from APIView, you can use the following command:
+
+`cli.bat apiview get-comments --review-id <ID> [--environment "production"|"staging"]`
```

Comment:
```suggestion
`cli.bat apiview get-comments --revision-id <ID> [--environment "production"|"staging"]`
```

Reply:
Nope, it should be `--review-id`. It was erroneously called revision_id previously.